.github/workflows/build-beta.yml

```
@@ -64,3 +64,7 @@ jobs:
           role-to-assume: ${{ vars.AWS_IAM_ROLE }}
           s3-bucket: ${{ vars.AWS_S3_BUCKET }}/${{ github.event.repository.name }}/${{ github.run_id }}/builds-beta
           path: builds
+
+      - if: failure()
+        name: canceling workflow
+        uses: Metamask/cancel-action@main
```

```suggestion
      - if: failure()
        name: Cancel failed workflow
        uses: MetaMask/cancel-action@main
```

We should also version this workflow, so we can do `MetaMask/cancel-action@v1` instead.

---

Sure, will pin to SHA as that is best practice over version numbers which can be supply chain attacked.